concrete-core/src/backends/core/implementation/engines/ggsw_ciphertext_discarding_conversion.rs

```diff
@@ -0,0 +1,238 @@
+use crate::backends::core::implementation::engines::CoreEngine;
+use crate::backends::core::implementation::entities::{
+    FourierLweBootstrapKey32, FourierLweBootstrapKey64, GgswCiphertext32, GgswCiphertext64,
+    GgswCiphertextComplex64,
+};
+use crate::backends::core::private::crypto::bootstrap::{
+    FourierBootstrapKey as ImplFourierBootstrapKey, StandardBootstrapKey,
```

Comment:
Here it seems to me you can just import the FourierBootstrapKey without renaming it?

Reply:
As above, I proceeded as such because it is how it was done in another similar function somewhere. I don't see any point in doing so but I wanted to preserve notations across the crate.

Reply:
Works for me: either we do it for the 2 of them or for none :smile: Then let's go for the two of them :wink:

Reply:
Will do!